app/src/main/res/layout/journey_info_cell.xml

```diff
@@ -84,20 +89,16 @@
 
             <TextView
                 android:id="@+id/carrier"
```

Comment:
Could we change this to operator to match what we're calling it elsewhere please